ChangeLog

```diff
@@ -49,6 +49,8 @@ Features
      independently contributed again by Paul Sokolovsky.
    * Add support for key wrapping modes based on AES as defined by
      NIST SP 800-38F algorithms KW and KWP and by RFC 3394 and RFC 5649.
+   * Add support for different keys sizes to CTR_DRBG. Note: using keys
```

Comment:
Not “different key sizes”, just 128. Something like (@yanesca @Patater):

> Add support for 128-bit keys in CTR_DRBG. Note that using keys shorter than 256 bits limits the security of generated material to 128 bits.

Reply:
I'm more than happy to use your phrasing. I'll wait to see if there are additional comments from @Patater or @yanesca before updating to avoid multiple updates.

Reply:
I am happy with the wording here too.

Reply:
Thanks @yanesca  and @gilles-peskine-arm , changed to text suggested by Gilles.